vault-handler/replicas-3/set_command.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: vault-handler
+  namespace: vault
+spec:
+  template:
+    spec:
+      containers:
+        - name: vault-handler
+          args: ['unseal']
```

Comment:
we need to add "--leader-only" flag,
`args:` ['unseal', '--leader-only']`

Reply:
Hey @jokestax, 
Do we need to add that option for both "versions" (single and three replicas)?

Reply:
The current one in `main` doesn't seem to have it @jokestax: https://github.com/konstructio/manifests/blob/main/vault-handler/replicas-3/job.yaml

Reply:
sorry guys I missed this, yes we don't require that in replicas3,only replicas one should be good enough